src/main/java/jalau/at18/katas/highestlowest/alvaro/Main.java

```diff
@@ -0,0 +1,38 @@
+package jalau.at18.katas.highestlowest.alvaro;
+
+public class Main {
+
+    public static void main(String[] args) {
+        String inputNumbers = "1 2 3 4 -5 200 -500";
+        int[] num;
+        String[] split = inputNumbers.split(" ");
+        num = new int[split.length];
+        for (int ind = 0; ind < split.length; ind++) {
+            num[ind] = Integer.parseInt(split[ind]);
+        }
+        compareGetHighestNumber(num);
+        compareGetLowestNumber(num);
+
+    }
+    public static int compareGetHighestNumber(int[]split) {
+        int max = 0;
+        for (int valor: split) {
```

Comment:
Hi @junior0123 , please use english for all your variable, method and class names